Composer/packages/client/src/components/CreationFlow/v2/TemplateDetailView.tsx

```diff
@@ -3,15 +3,69 @@
 // Licensed under the MIT License.
 
 /** @jsx jsx */
-import { jsx } from '@emotion/core';
+import { BotTemplate } from '@bfc/shared';
+import { css, jsx } from '@emotion/core';
+import formatMessage from 'format-message';
 import React from 'react';
 import ReactMarkdown from 'react-markdown';
 
+import composerIcon from '../../../images/composerIcon.svg';
+
+const templateTitleContainer = css`
+  width: 100%;
+  padding-right: 2%;
+  height: fit-content
+  overflow: hidden;
+  flex-grow: 1;
+  float: left;
+  word-break: break-all;
+`;
+
+const templateTitle = css`
+  position: relative;
+  bottom: 18px;
+  font-size: 19px;
+  font-weight: 550;
+  margin-left: 10px;
+`;
+
+const templateVersion = css`
+  position: relative;
+  font-size: 12px;
+  font-weight: 100;
+  display: block;
+  left: 55px;
+  width: fit-content;
+  bottom: 20px;
+`;
+
 type TemplateDetailViewProps = {
-  templateId: string;
+  template?: BotTemplate;
   readMe: string;
 };
 
 export const TemplateDetailView: React.FC<TemplateDetailViewProps> = (props) => {
-  return <ReactMarkdown>{props.readMe}</ReactMarkdown>;
+  // readMe comes from npm where first line is the template title.
+  // Composer formats and displays its own template title and strips out the first line to avoid redundent titles
+  const getStrippedReadMe = () => {
+    const lines = props.readMe.split('\n');
```

Comment:
maybe just do this all on one line?

return props.readMe.split(/\n/).splice(0,1).join('\n')

Reply:
Actually, this will only work if the title is the ABSOLUTE first line of the returned readme. 

We may want to do a regular expression, something like `props.readMe.replace(/^# (.*?)\n/)`. which should replace the first instance of `# (something)\n`

Reply:
handled